extensions/wikia/AdEngine/js/spec/AdEngineRunner.spec.js

```diff
@@ -4,6 +4,7 @@ describe('ext.wikia.adEngine.adEngineRunner', function () {
 	function noop() {
 	}
 
+	// TODO tests
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app